.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**I confirm:**
+<!-- To mark a task as complete, use [x] -->
```

Comment:
Not sure reporters will understand the specifics of GitHub flavored Markdown

Reply:
That's why I put the reminder there, to mark a task as complete put a x in the box. So reporters who don't know GitHub Flavoured Markdown can use the check-boxes too.

Reply:
If you don't like the check-boxes, I can of course convert that part to a bulleted or numbered list.

So let me know what you like best. :+1:

Reply:
The thing is: while editing the ticket's message, the check boxes are still in Markdown, so the magic of conversion to widgets is not active. Plus, we have no way to force reporters to check the boxes by some validation process, so they will typically ignore these warnings (experience from git-scm.com project).

Reply:
I see your points, I think we'll be better of with just a plain list. I've converted the document to use a list instead of the previous check boxes.